examples/torch/ddpg_pendulum.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+"""
+This is an example to train a task with DDPG algorithm written in PyTorch.
+
+Here it creates a gym environment InvertedDoublePendulum. And uses a DDPG with
+1M steps.
+
+"""
+import gym
+import torch
+from torch.nn import functional as F  # NOQA
+
+from garage.envs import normalize
+from garage.experiment import LocalRunner, run_experiment
+from garage.np.exploration_strategies import OUStrategy
+from garage.replay_buffer import SimpleReplayBuffer
+from garage.tf.envs import TfEnv
+from garage.torch.algos import DDPG
+from garage.torch.modules import MLPModule
+from garage.torch.policies import DeterministicPolicy
+from garage.torch.q_functions import ContinuousNNQFunction
+
+
+def run_task(*_):
+    """Set up environment and algorithm and run the task."""
+    with LocalRunner() as runner:
+        env = TfEnv(normalize(gym.make('InvertedDoublePendulum-v2')))
+
+        action_noise = OUStrategy(env.spec, sigma=0.2)
+
+        policy_module = MLPModule(
+            input_dim=env.spec.observation_space.flat_dim,
+            output_dim=env.spec.action_space.flat_dim,
+            hidden_sizes=[64, 64],
+            hidden_nonlinearity=F.relu,
+            output_nonlinearity=torch.tanh)
+        policy = DeterministicPolicy(env.spec, policy_module)
+
+        input_dim = (env.spec.observation_space.flat_dim +
```

Comment:
this is a real pain and error-prone. i'd prefer if we define `ContinuousMLPQFunction` which infers this directly from the env_spec.